Sources/SwiftRefactor/ConvertComputedPropertyToZeroParameterFunction.swift

```diff
@@ -0,0 +1,133 @@
+  public static func refactor(syntax: VariableDeclSyntax, in context: Void) -> FunctionDeclSyntax? {
+    guard syntax.bindings.count == 1,
+      let binding = syntax.bindings.first,
+      let identifierPattern = binding.pattern.as(IdentifierPatternSyntax.self)
+    else { return nil }
+
+    var statements: CodeBlockItemListSyntax
+
+    guard let typeAnnotation = binding.typeAnnotation,
+      var accessorBlock = binding.accessorBlock
+    else { return nil }
+
+    var effectSpecifiers: AccessorEffectSpecifiersSyntax?
+
+    switch accessorBlock.accessors {
+    case .accessors(let accessors):
+      guard accessors.count == 1, let accessor = accessors.first,
+        accessor.accessorSpecifier.tokenKind == .keyword(.get), let codeBlock = accessor.body
+      else { return nil }
+      effectSpecifiers = accessor.effectSpecifiers
+      statements = codeBlock.statements
+      let accessorSpecifier = accessor.accessorSpecifier
+      statements.leadingTrivia =
+        accessorSpecifier.leadingTrivia + accessorSpecifier.trailingTrivia.droppingLeadingWhitespace
+        + codeBlock.leftBrace.leadingTrivia.droppingLeadingWhitespace
+        + codeBlock.leftBrace.trailingTrivia.droppingLeadingWhitespace
+        + statements.leadingTrivia
+      statements.trailingTrivia += codeBlock.rightBrace.trivia.droppingLeadingWhitespace
+      statements.trailingTrivia = statements.trailingTrivia.droppingTrailingWhitespace
+    case .getter(let codeBlock):
+      statements = codeBlock
+    }
+
+    let returnType = typeAnnotation.type
+
+    var returnClause: ReturnClauseSyntax?
+    let triviaAfterSignature: Trivia
+
+    if !returnType.isVoid {
+      triviaAfterSignature = .space
+      returnClause = ReturnClauseSyntax(
+        arrow: .arrowToken(
+          leadingTrivia: typeAnnotation.colon.leadingTrivia,
+          trailingTrivia: typeAnnotation.colon.trailingTrivia
+        ),
+        type: returnType
+      )
+    } else {
+      triviaAfterSignature = typeAnnotation.colon.leadingTrivia + typeAnnotation.colon.trailingTrivia
+    }
+
+    accessorBlock.leftBrace.leadingTrivia = accessorBlock.leftBrace.leadingTrivia.droppingLeadingWhitespace
+    accessorBlock.rightBrace.trailingTrivia = accessorBlock.rightBrace.trailingTrivia.droppingTrailingWhitespace
+
+    let body = CodeBlockSyntax(
+      leftBrace: accessorBlock.leftBrace,
+      statements: statements,
+      rightBrace: accessorBlock.rightBrace
+    )
+
+    var parameterClause = FunctionParameterClauseSyntax(parameters: [])
+    parameterClause.trailingTrivia = identifierPattern.identifier.trailingTrivia + triviaAfterSignature
+
+    let functionEffectSpecifiers = FunctionEffectSpecifiersSyntax(
+      leadingTrivia: effectSpecifiers?.leadingTrivia,
+      asyncSpecifier: effectSpecifiers?.asyncSpecifier,
+      throwsClause: effectSpecifiers?.throwsClause,
+      trailingTrivia: effectSpecifiers?.trailingTrivia.droppingTrailingWhitespace
+    )
```

Comment:
A syntax node doesn’t have any inherent trivia, `effectsSpecifiers.leadingTrivia` will be the leading trivia of the first token within that tree. Since that will most likely be the `asyncSpecifier`, this will be duplicating the leading trivia of `asyncSpecifier` (same for `trailingTrivia`).

```suggestion
    let functionEffectSpecifiers = FunctionEffectSpecifiersSyntax(
      asyncSpecifier: effectSpecifiers?.asyncSpecifier,
      throwsClause: effectSpecifiers?.throwsClause
    )
```